docs/source/advanced/multi_gpu.rst

```diff
@@ -239,6 +239,20 @@ Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
     to be in "exclusive mode", such that only one process at a time can access them.
     For more details see the :doc:`trainer guide <../common/trainer>`.
 
+
+Select Torch Distributed Backend
```

Comment:
the other titles don't use capitalization

Reply:
removed !